package.json

```diff
@@ -21,7 +21,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "connman-api": "git+https://github.com/Doodle3D/connman-api.git",
+    "connman-api": "git+https://github.com/pcarranzav/connman-api.git#5bae8c4d15066ac36a57069c2003901e5dee398d",
```

Comment:
This should be changed to the corresponding connman-api version once the changes from [#7](https://github.com/Doodle3D/connman-api/pull/7) are published to the npm registry.